frontend/app/work_packages/controllers/details-tab-overview-controller.js

```diff
@@ -137,9 +137,42 @@ module.exports = function($scope,
   (function setupWorkPackageProperties() {
     var otherAttributes = WorkPackagesOverviewService.getGroupAttributesForGroupedAttributes('other', $scope.groupedAttributes);
 
-    angular.forEach($scope.workPackage.props.customProperties, function(customProperty) {
-      this.push(customProperty);
-    }, otherAttributes);
+    function getFormat(workPackage, prop, propName) {
+      return workPackage.props[propName] ?
+              workPackage.props[propName].format :
+              prop.type.toLowerCase();
+    }
+
+    function getValue(workPackage, propName) {
+      if (workPackage.props[propName]) {
+        return workPackage.props[propName].raw;
+      }
+      if (workPackage.embedded[propName]) {
+        return workPackage.embedded[propName].props.id;
+      }
+      return null;
+    }
+
+    function getCustomProperties(workPackage) {
+      return _.compact(_.map(
```

Comment:
@NobodysNightmare '_' is the god object of a lib called lodash that does all that fancy ruby stuff you have like .flatten, .compact, .map, .reduce. This var is global and hound does not see it. I am sure there is some weird /* iddqd */ comment that would make it shut up